fiftyone/zoo/datasets/base.py

```diff
@@ -847,6 +848,162 @@ def _download_and_prepare(self, dataset_dir, scratch_dir, split):
+
+        subset = foz.load_zoo_dataset(
+            "open-images",
+            split="validation",
+            label_types=["detections", "relationships"],
+            classes=["Fedora", "Piano"],
+            max_samples=50,
+        )
+
+        session.dataset = subset
+
+    Dataset size
+        561 GB
+
+    Source
+        https://storage.googleapis.com/openimages/web/index.html
+
+    Args:
+        label_types (None): a list of label types to load. Supported values are
+            ``("detections", "classifications", "relationships", "segmentations")``.
+            By default, all label types are loaded. Note that not every sample
+            will include each label type. If ``max_samples`` and
+            ``label_types`` are both specified, then every sample will include
+            the specified label types
+        classes (None): a list of strings specifying required classes to load.
+            Only samples containing at least one instance of a specified
+            classes will be downloaded. See available classes with
+            :meth:`fiftyone.utils.openimages.get_classes()`
+        attrs (None): a list of strings for relationship attributes to load
+        max_samples (None): a maximum number of samples to import per split. By
+            default, all samples are imported
+        seed (None): a random seed to use when shuffling
+        shuffle (False): whether to randomly shuffle the order in which the
+            samples are imported
+        image_ids (None): a list of specific image IDs to load. The IDs can be
+            specified either as ``<split>/<image-id>`` or ``<image-id>``
+        image_ids_file (None): the path to a newline separated text, JSON, or
+            CSV file containing a list of image IDs to load. The IDs can be
+            specified either as ``<split>/<image-id>`` or ``<image-id>``. If
+            ``image_ids`` is provided, this parameter is ignored
+        num_workers (None): the number of processes to use when downloading
+            individual images. By default, ``multiprocessing.cpu_count()`` is
+            used
+        version ("v6"): string indicating the version of Open Images to
+            download. Currently only Open Images V6 is supported.
+    """
+
+    def __init__(
+        self,
+        label_types=None,
+        classes=None,
+        attrs=None,
+        max_samples=None,
+        seed=None,
+        shuffle=None,
+        image_ids=None,
+        image_ids_file=None,
+        num_workers=None,
+        version="v6",
+    ):
+
+        self.label_types = label_types
+        self.classes = classes
+        self.attrs = attrs
+        self.max_samples = max_samples
+        self.seed = seed
+        self.shuffle = shuffle
+        self.image_ids = image_ids
+        self.image_ids_file = image_ids_file
+        self.num_workers = num_workers
+        self.version = version
+
+    @property
+    def name(self):
+        return "open-images"
```

Comment:
So for other datasets with versions like `coco-2014` and `coco-2017`, we put the "version" directly in the name of the zoo dataset.

In this case, it is currently undocumented, but users of Open Images can get a particular version (when more are supported) via the syntax `load_zoo_dataset("open-images", ..., version="v6")`, which is a 

Perhaps you should explicitly go with `OpenImagesV6Dataset` and `open-images-v6` in this class definition, for consistency with how we're handling other datasets? Another reason this makes sense is that the docstring of this class is specific to V6; those details will change for other versions of the dataset, suggesting that dedicated classes are indeed the right approach.

(Currently the content from the class docstring is manually copied over to generate the zoo docs, but someday I'd like to automate that.)

The underlying utils methods can still accept a `version` argument, I'm just referring specifying to the ZooDataset class here.